easybuild/easyconfigs/m/MCR/MCR-R2021b.1.eb

```
@@ -0,0 +1,21 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'MCR'
+version = 'R2021b'  # runtime version 9.11
+local_update = '1'
+versionsuffix = '.%s' % local_update
+
+homepage = 'https://www.mathworks.com/products/compiler/mcr/'
+description = """The MATLAB Runtime is a standalone set of shared libraries
+ that enables the execution of compiled MATLAB applications
+ or components on computers that do not have MATLAB installed."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://ssd.mathworks.com/supportfiles/downloads/%%(version)s/Release/%s/deployment_files/'
+               'installer/complete/glnxa64/' % local_update]
+sources = ['MATLAB_Runtime_%(version)s_glnxa64.zip']
```

I think this has to be:
```suggestion
sources = ['MATLAB_Runtime_%%(version)s_Update_%s_glnxa64.zip' % local_update]
```
?